src/Compilers/VisualBasic/Portable/BoundTree/BoundNodes.xml

```diff
@@ -1017,7 +1017,7 @@
     <!-- Parameter receiving the value. -->
     <Field Name="Parameter" Type="ParameterSymbol" Null="disallow"/>
     <!-- Expression representing the value. -->
-    <Field Name="Value" Type="BoundExpression" Null="disallow"/>
+    <Field Name="Value" Type="BoundExpression" Null="allow"/>
```

Comment:
That source construct does not have an `= Value` clause to bind, so why would you need to modify this bound node?

---
In reply to: [72648837](https://github.com/dotnet/roslyn/pull/12629#discussion_r72648837) [](ancestors = 72648837)